### DISCUSSION TITLE
Atualizações da ESRI Portugal

### DISCUSSION THREAD

**Discussion by pmoura11** [dssg-pt/covid19pt-data#783] [Geral]
Parece-me que a ESRI Portugal está a deixar de fazer manutenção regular dos dados COVID-19.
Alguém sabe se existe alguma informação oficial ou oficiosa para este comportamento?

**Comment by davipt** [comment]
Tem acontecido estar 1 ou 2 dias sem actualização e depois corrigem os dados em falta - o que não aconteceu para o dia 25, que continue em falta.

Assumo que no pior dos casos actualizem na sexta-feira, com os dados de concelhos. Vamos ver.

**Reply by pmoura11** [reply]
Neste exato momento (06/05) a informação disponível na ESRI é de 02/05.
Desconfinamos nós, enquanto eles confinam os dados!
Nunca entendi esta cedência de informação em exclusividade à ESRI, mas se esta entidade não se sente à altura do desafio, a DGS (ou a Closer, ou lá quem tem os dados) deveriam escolher outra entidade para divulgar publicamente, para que isto não morra a meio...
Sei que isto nada tem a haver com a DSSG, mas fica aqui o desabafo!
Continuação de bom trabalho para todos...

**Reply by davipt** [reply]
De vez em quando meto um tweet com menção à ESRI e DGS. Não faço ideia se ajuda ou não, mas vi que já fizeram actualização dos dados de 3, 4, e 5 de maio - 25 de abril continua em falta.

Assim que publiquem o PDF de hoje já actualizo os dados de hoje com a actualização dos dias anteriores. Até já.

**Reply by davipt** [reply]
Está actualizado, inclusive os dados de hoje :) Podes confirmar?
